rank/components/H.tsx

```diff
@@ -2,8 +2,18 @@ import { FC } from 'react'
 
 const H2: FC = ({ children }) => {
   return (
-    <h2 className="text-xl mt-1 mb-2 border-t border-gray-200">{children}</h2>
+    <h2 className="text-3xl mt-1 mb-4 border-l-4 pl-2 border-blue-300">
+      {children}
+    </h2>
   )
 }
 
-export { H2 }
+const H3: FC = ({ children }) => {
```

Comment:
Out of interest, what's the benefit for creating a set of `H` components here instead of just setting these styles globally in `/styles`, eg:

```
h2 {
  @apply text-3xl mt-1 mb-4 border-l-4 pl-2 border-blue-300;
}

h3 {
  @apply text-lg mt-1 mb-2 border-l-4 pl-2 border-pink-300;
}
```

afaik we don't use any headings which aren't intended to have these styles applied to them

Reply:
Basically so that you can use them if you want, but not if you don't want. Setting global styles always makes this incredible faffy, and lands up with loads of overrides. 

We could create a style `.h2` sa above - and then apply that to anything, which maybe fits the [utility first](https://tailwindcss.com/docs/utility-first) mentality, but I've gone for the [extracting components](https://tailwindcss.com/docs/extracting-components) method.